data/themes/darktable.css

```diff
@@ -1312,42 +1166,37 @@ dialog scrollbar
   font-weight: bold;
 }
 
-#preferences_notebook scrolledwindow #pref_section  /* top margin in sub-category title in main content */
+#preferences_notebook scrolledwindow #pref_section :not(:first-child) /* top margin in sub-category title in main content */
```

Comment:
Would it be possible to do something similar for `section_label` so that it automatically look good even at the top of a box without requiring this line in the code?
`gtk_style_context_add_class(context, "section_label_top");`

Reply:
This is only for when the section label is at the top of an iop rather than when it is the first section label _in_ the iop (which might be after some other controls). But I suspect it could be achieved with pure CSS.

Reply:
When the section label is the first _child_ within a box. Could be that the box is on a notebook page, doesn't matter; if the first thing within that box is a section label, it should get special treatment.
But if there first is something else (slider, whatever) and _then_ a section label; even though it might be the first _section label_ it should just be treated the same as any following section labels.

:first-child sounds like it should do the right thing...

Reply:
could be, yes

Reply:
first-child is exactly for that: setting a first child of a widget differently than following ones. Here, with not, I just apply to all except first child. About @section label, that should come. Many things remain to do.